apps/docs/playwright.config.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
```

Comment:
I think this should be in the config package so that it's shared across all apps.